src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/GenericXmlSecurityKeyIdentifierClause.cs

```diff
@@ -0,0 +1,79 @@
+using System.IdentityModel.Tokens;
+using System.ServiceModel;
```

Comment:
Question only: Is adding `using System.ServiceModel` part of the consolidation of code between System.ServiceModel and System.IdentityModel?